db/schema.rb

```diff
@@ -137,10 +137,7 @@
   create_table "systems", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
     t.string "login"
     t.string "password"
-    t.string "guid"
-    t.string "secret"
     t.string "hostname"
-    t.string "target"
```

Comment:
Target was used in the zypper service XML. There's a small chance it would be needed for https://trello.com/c/fBNlMlg0/710 -- although probably it was used only before SLE12 came along.

Reply:
I'm thinking that it would be better to remove it now and then add it when we're using it. Keeping it may influence how we face that feature, removing it now may create a "blank slate" where a developer has the freedom to find a better solution.

Reply:
Fair enough.

Reply:
But thanks for your input. I was not aware of that card or if we would even need these fields in the future.